runtime/redirector/forwarders.m4

```diff
@@ -265,29 +265,29 @@ _X(JVM_PrintStackTrace,JNICALL,true,jobject ,jint arg0, jint arg1, jint arg2)
 _X(JVM_SetField,JNICALL,true,jobject ,jint arg0, jint arg1, jint arg2, jint arg3)
 _X(JVM_SetPrimitiveField,JNICALL,true,jobject ,jint arg0, jint arg1, jint arg2, jint arg3, jint arg4, jint arg5)
 _X(JVM_SetNativeThreadName,JNICALL,true,void ,jint arg0, jobject arg1, jstring arg2)
-_IF([(!defined(J9VM_JAVA9_BUILD)) || (J9VM_JAVA9_BUILD < 148)],
+_IF([J9VM_JAVA9_BUILD < 148],
 	[_X(JVM_DefineModule,JNICALL,false,jobject ,JNIEnv arg0, jobject arg1, jstring arg2, jstring arg3, jobjectArray arg4)])
-_IF([defined(J9VM_JAVA9_BUILD) && (J9VM_JAVA9_BUILD >= 148) && (J9VM_JAVA9_BUILD < 156)],
+_IF([(148 <= J9VM_JAVA9_BUILD) && (J9VM_JAVA9_BUILD < 156)],
```

Comment:
Constants on the left is only desired for ==. All the other constants here are on the right.

Reply:
Any response to this @keithc-ca ?

Reply:
With my mathematical background, I find it easier to read 'between' conditions such as this when expressed as `((min <= x) && (x < max))`; it's as close as we can get to 'min <= x < max'.